docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -1331,6 +1331,30 @@ public void register(RegistrationContext context) {
 
 <1> The argument is the bindings source class.
 
+=== `InstanceHandle.close()` Behavior
```

Comment:
Since we're talking about the spec compatibility shouldn't we use `jakarta.enterprise.inject.Instance.Handle` in the title?

Reply:
It's the same thing, but yeah, makes it less confusing. Will do.

Reply:
Done.